UOP1_Project/Assets/Scripts/Editor/ReplaceTool.cs

```diff
@@ -137,12 +137,17 @@ private void ReplaceSelectedObjects(GameObject[] objectToReplace, GameObject rep
 		{
 			var go = objectToReplace[i];
 			Undo.RegisterCompleteObjectUndo(go, "Saving game object state");
-			var inst = Instantiate(replaceObject, go.transform.position, go.transform.rotation, go.transform.parent);
-			inst.transform.localScale = go.transform.localScale;
-			Undo.RegisterCreatedObjectUndo(inst, "Replacement creation.");
+			//var inst = Instantiate(replaceObject, go.transform.position, go.transform.rotation, go.transform.parent);
```

Comment:
Not really necessary to keep this around as a comment if it's been replaced.

Reply:
agreed